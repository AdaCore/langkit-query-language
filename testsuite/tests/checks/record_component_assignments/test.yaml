driver: 'checker'
rule_name: record_component_assignments
project: 'prj.gpr'
