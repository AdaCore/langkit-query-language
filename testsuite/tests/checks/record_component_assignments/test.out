assign.adb:14:4: rule violation: record component assignment
14 |    O1.Comp1 := 1;   --  FLAG
   |    ^^^^^^^^^^^^^^

